lnd_methods/offchain/estimate_route_fee.js

```diff
@@ -0,0 +1,83 @@
+const asyncAuto = require('async/auto');
+const {returnResult} = require('asyncjs-util');
+
+/**
+ * Estimate routing fees based on an invoice.
+ *
+ * Requires `offchain:read` permission
+ *
+ * This method is not supported before LND 0.18.4
+ *
+ *
+ @argument
+ {
+    lnd: <Authenticated LND API Object>
+    request: <BOLT 11 Payment Request String>
+    timeout: <Optional Timeout in Milliseconds Number>
+  }
+
+  @returns via cbk or Promise
+  {
+    mtoken: <Route Fee Millitokens String>
```

Comment:
i think this should be `fee_mtokens` and include a `fee` for the sats value, or just `mtokens` instead of `mtoken`

Reply:
Sounds good. To clarify for posterity, I'm going to eliminate `fee_mtokens` in favor of `fee` measured in sats and returns as a number not a string.